source/_components/device_tracker.asuswrt.markdown

```diff
@@ -35,6 +35,7 @@ Configuration variables:
 - **username** (*Required*: The username of an user with administrative privileges, usually `admin`.
 - **password** (*Optional*): The password for your given admin account (use this if no SSH key is given).
 - **protocol** (*Optional*): The protocol (`ssh` or `telnet`) to use. Defaults to `ssh`.
+- **port** (*Optional*): SSH port to use. Defaults to 22.
```

Comment:
Nit: It may be nice to have `22` in backticks like we do for `192.168.1.1` but I don't feel strongly